providers/dns/pdns/pdns.go

```diff
@@ -173,7 +173,7 @@ func (d *DNSProvider) Present(domain, token, keyAuth string) error {
 		return fmt.Errorf("pdns: %w", err)
 	}
 
-	if d.apiVersion < 1 {
+	if d.apiVersion < 1 || (zone.Kind != "Master" && zone.Kind != "Slave") {
```

Comment:
Can you extract the logic into a helper function, and invert the logic here?

Something like

```go
if supportsZonetransfer(d.apiVersion, zone.Kind) {
   	err = d.notify(zone.URL)
	if err != nil {
		return fmt.Errorf("pdns: %w", err)
	}
}
return nil
```